packages/apps/spaces/src/graphQL/hooks.ts

```diff
@@ -0,0 +1,86 @@
+import { gql } from '@apollo/client';
```

Comment:
Please add a README.md to the project for the generation of the graphQL files

Reply:
https://github.com/openline-ai/openline-customer-os/issues/875